src/main/resources/infusionsoft/js/ps2ps_contact_event.js

```diff
@@ -10,7 +10,7 @@ function transform(message) {
 
 function toCanonical(event) {
     var canonical = {};
-    var eventData = event.event_data;
+    var eventData = event.source_data;
```

Comment:
event though the field is literally named "eventData", flagship's JSON declaration calls it "source_data", so that's how it is coming across the wire...